packages/ui/src/ui/common/thor/utils.ts

```diff
@@ -0,0 +1,32 @@
+//@ts-ignore
+import ypath from '@ytsaurus/interface-helpers/lib/ypath';
+import {appendInnerErrors} from '../../utils/errors';
+
+export function convertToNumber<T extends number | undefined>(
+    value: any,
+    defaultValue?: T,
+): number | T {
+    if (value === null) return defaultValue as T;
+    const res = Number(value);
+    return isNaN(res) ? (defaultValue as T) : res;
+}
+
+export const getNumber = function <T extends number | undefined>(
```

Comment:
It is better to have some tests for `getNumber` too. It looks like an exception will be never thrown from the function.